### PR TITLE
[Actions] Document authentication for service containers

### LINKED ISSUE #29914: Specifying credentials for service containers
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/using-containerized-services/about-service-containers

### What part(s) of the article would you like to see updated?

The docs don't show how to specify credentials when using service containers. Credentials may be necessary to pull images from private registries or to authenticate with DockerHub in order to increase rate limits.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ndench**
Document how to authenticate with private registries and DockerHub when pulling images  for service containers.

### Why:

The documentation for service containers doesn't currently show how to specify credentials.
However, there's a [blog post from 2020 showing how to do it](https://github.blog/changelog/2020-09-24-github-actions-private-registry-support-for-job-and-service-containers/).

Closes: #29914

### What's being changed (if available, include any code snippets, screenshots, or gifs):

The docs don't show how to specify credentials when using service containers. Credentials may be necessary to pull images from private registries or to authenticate with DockerHub in order to increase rate limits.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/actions/using-containerized-services/about-service-containers.md [hunk 1]
```diff
@@ -120,6 +120,36 @@ jobs:
 
 {% endraw %}
 
+## Authenticating with image registries
+
+You can specify credentials for your service containers in case you need to authenticate with an image registry. This allows you to use images from private registries or to [increase your DockerHub rate limit](https://www.docker.com/increase-rate-limits/).
+
+Here’s an example of authenticating with DockerHub and GitHub Container Registry:
```
Comment (hubwriter) [inline content/actions/using-containerized-services/about-service-containers.md:127]: Use variables for the GH Container registry:

```suggestion
Here’s an example of authenticating with Docker Hub and the {% data variables.product.prodname_dotcom %} {% data variables.product.prodname_container_registry %}:
```

**Review by hubwriter — CHANGES_REQUESTED** [review]
@ndench - Great change, many thanks for adding this! 👍 

Just one suggestion - which you can click to commit.

**Review by hubwriter — APPROVED** [review]

**Comment by hubwriter** [comment]
Merging this now

**Comment by s3m13** [comment]
https://pin.it/7I9ZoqhgF

**Pull request merged**